civictechprojects/static/css/partials/_SubHeader.scss

```diff
@@ -17,21 +16,17 @@
 
   background-color: $color-background-light;
   height: $header-height;
-  width: 100%;
-  //padding: 0 5% 0 5%;
-
-  //background-color: $color-background-default;
-  padding: 5px 5% 5px 5%;
-
+  padding: 0 5px 0 5px;
   margin-bottom: 5px;
+  justify-content: space-between;
 }
 
 .SubHeader-rightContent {
   display: none;
   flex-direction: row;
   height: 100%;
+  padding-right: 115px;  
```

Comment:
This is putting extra space on the right side.

![image](https://user-images.githubusercontent.com/322346/55973780-1bc17580-5c3b-11e9-966d-ffd370717115.png)